components/mastodon/sources/new-status-favorited/new-status-favorited.mjs

```diff
@@ -0,0 +1,49 @@
+import common from "../common/base.mjs";
+
+export default {
+  ...common,
+  key: "mastodon-new-status-favorited",
+  name: "New Status Favorited",
+  description: "Emit new event when the specified status is favorited",
```

Comment:
you could also add link to the docs of the currently used endpoint on this source